### DISCUSSION TITLE
When you click to delete a collection, add the name of the collection to the confirmation message

### DISCUSSION THREAD

**Discussion by sxsws** [directus/directus#13473] [Feature Requests]
It is reassuring to see 'do you want to delete name_of_collection' when you click to delete it, rather than just seeing 'do you want to delete this collection'.

Just like it does when you delete a field, it mentions the name of the field, I think it should do the same with the collection. Is the name available at the time of clicking on the delete?

**Comment by benhaynes** [comment]
I agree, would be a great simple pull request!

**Reply by sxsws** [reply]
done! As I'm so new to node, I'm not sure how to build to check it, but I could just submit a pr? It's only the vue file and some translation files!

**Reply by benhaynes** [reply]
Yeah, that should work. Thank you!

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
